packages/ketcher-react/src/script/ui/views/modal/components/toolbox/Bond/Bond.tsx

```diff
@@ -45,26 +93,79 @@ const Bond = (props: Props) => {
       buttons={['Cancel', 'OK']}
       withDivider
     >
-      <Form schema={bondSchema} init={rest} {...formState}>
+      <Form
+        schema={bondSchema}
+        init={rest}
+        {...formState}
+        customValid={customValid}
+      >
         <Field
           name="type"
           component={Select}
           options={getSelectOptionsFromSchema(bondProps.type)}
+          disabled={isCustomQuery}
+          formName="bond-properties"
         />
         <Field
           name="topology"
           component={Select}
           options={getSelectOptionsFromSchema(bondProps.topology)}
+          disabled={isCustomQuery}
+          formName="bond-properties"
         />
         <Field
           name="center"
           component={Select}
           options={getSelectOptionsFromSchema(bondProps.center)}
+          disabled={isCustomQuery}
+          formName="bond-properties"
         />
+        <div className={classes.customQueryWrapper}>
+          <CustomQueryField
```

Comment:
My proposal is to add data-testid for this in advance

Reply:
done